.github/workflows/pre-main.yaml

```diff
@@ -275,28 +273,11 @@ jobs:
       - name: Remove tarball(s) to save disk space.
         run: rm -f cnf-certification-test/*.tar.gz
 
-      - name: Run gradetool on the claim.json.
-        run: |
-          docker run \
-            --rm \
-            --volume "${GITHUB_WORKSPACE}"/generated_policy.json:/policy.json \
-            --volume "${GITHUB_WORKSPACE}"/cnf-certification-test/claim.json:/claim.json \
-            ${REGISTRY}/${GRADETOOL_IMAGE_NAME}:${GRADETOOL_IMAGE_TAG} \
-              --OutputPath results.txt \
-              --policy /policy.json \
-              --results /claim.json \
-              >"${GITHUB_WORKSPACE}"/results.txt
-          docker system prune --volumes -f
+      - name: Build the TNF tool
+        run: make build-tnf-tool
```

Comment:
It can be done in one step: `make build-tnf-tool check-results`

Reply:
Thanks for the suggestion but I prefer to leave it as two logical steps in the workflow. This way if it fails it's easier to identify if it was the tool compilation or the results verification. Also, it mirrors better the container job, which does not use the `make` to run the tool.